sdk/storage/azure-storage-files-datalake/CMakeLists.txt

```diff
@@ -64,7 +64,6 @@ target_include_directories(
     PUBLIC
       $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/inc>
       $<INSTALL_INTERFACE:include>
-      ${azure-storage-blobs-cpp_INCLUDE_DIRS}
```

Comment:
Don't remove.

Reply:
duplicate, discuss in another conversation